### PR TITLE
LPS-86087 Search Insights Widget: Explain + Search Response String

### PR DESCRIPTION
**Pull request by arboliveira**

### CODE REVIEW

**Comment by arboliveira** [comment]
ci:test:sf

**Comment by arboliveira** [comment]
ci:test:sf

**Comment by arboliveira** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -41,6 +41,8 @@ display-scope-facet=Display Site Facet
 display-selected-result-in-context=Display Selected Result in Context
 display-selected-result-in-context-help=Display the full context of a selected result when available.
 display-user-facet=Display User Facet
+explain=Explain
+explain-help=When enabled, a search query also returns explanation for each hit on how its score was computed.
```
Comment (rbohl) [inline modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: >  explain-help=Return a score explanation for each hit returned by the search engine.

How does that sound?

Comment (arboliveira) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: https://github.com/arboliveira/liferay-portal/pull/638/commits/7a21a56d3e0d19a055663966ff50c9c11a54fbf7

**Comment by arboliveira** [comment]
ci:test:search:norebase

**Comment by xbrianlee** [comment]
ci:test:search

**Comment by arboliveira** [comment]
ci:test:search:norebase

**Comment by arboliveira** [comment]
ci:test:search:norebase

**Comment by arboliveira** [comment]
ci:test:search:norebase

**Comment by arboliveira** [comment]
ci:test:sf:norebase

**Pull request closed without merging**